dvc/tree/gdrive.py

```diff
@@ -510,19 +509,29 @@ def _path_to_item_ids(self, path, create, use_cache):
         )
 
     def _get_item_id(self, path_info, create=False, use_cache=True, hint=None):
-        assert path_info.bucket == self._bucket
-
-        item_ids = self._path_to_item_ids(path_info.path, create, use_cache)
-        if item_ids:
-            return min(item_ids)
+        if isinstance(path_info, GDriveURLInfo):
```

Comment:
not sure I understand this logic, to be honest ... GDriveURLInfo can be pretty much anything, not only `grive://ID` - it can be `gdrive://root/some/path/here`, etc

Reply:
In this case, the logic is definitively limited. I thought `gdrive://ID` was the unique form of gdrive url.

Reply:
@shcheklein is the `gdrive://` scheme a DVC convention? I didn't find any reference in the API documentation or on the web.

Reply:
yes, `gdrive://`, it is described here - https://dvc.org/doc/user-guide/setup-google-drive-remote#url-format.